runtime/lua/vim/lsp/buf.lua

```diff
@@ -129,6 +130,11 @@ local function range_from_selection()
   local start_col = start[3]
   local end_row = end_[2]
   local end_col = end_[3]
+  if mode == 'V' then
+    start_col = 1
+    local lines = api.nvim_buf_get_lines(0, end_row - 1, end_row, true)
+    end_col = #lines[1]
```

Comment:
uh, `start_col` is not head line of the selection.
(it is "start" of the selection)
and also, `end_col` is not tail line of the selection.

If we select lines from row 4 to row 6, it works right.
But when we select lines form row 6 to row 4, it does not work.

Reply:
it is considered in https://github.com/neovim/neovim/pull/22632/files#diff-ac41943ef406cefaa6a72f1e1b157d024e1977bb4ea52e3965318e181ea0effaR141-R151.
So maybe it should processed after them.